crates/bin/src/args.rs

```diff
@@ -298,14 +298,15 @@ impl Default for RateLimit {
 
 /// Strategy for installing the package
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, ValueEnum, EnumCount)]
+#[repr(u8)]
 pub enum Strategy {
     /// Attempt to download official pre-built artifacts using
     /// information provided in `Cargo.toml`.
-    CrateMetaData,
+    CrateMetaData = 0,
```

Comment:
This is actually implied, but if we're making it explicit: if we start at `1` it niches better, actually.